pxtcompiler/emitter/service.ts

```diff
@@ -216,6 +216,7 @@ namespace ts.pxtc {
             let hasParams = kind == SymbolKind.Function || kind == SymbolKind.Method
 
             let pkg: string = null
+            let pkgs: string[] = []
```

Comment:
don't eagerly allocate this array as it is most of the time not used

Reply:
Good point, I've changed this to be null by default.